### PR TITLE
More fixes on template page.

### PR DESCRIPTION
**Pull request by benkasminbullock**

### CODE REVIEW

File: content/3.0templating.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ parameter while executing the template, if you are wondering how we'd do that, t
 		}
 
 These are the three parts of using templates, first you need to create types like we have created the `Context` type, then we need to read the template, 
-then we need to use the components of that type in our template file. So what remains now is passing an object of that type while template execution.
+then we need to use the components of that type in our template file. So what remains now is passing an object of that type during template execution.
 
 Reading template:
 
-		templates, err = template.ParseFiles(allFiles...)
+		templates, err = template.ParseGlob(allFiles...)
```
Comment (thewhitetulip) [inline content/3.0templating.md]: @benkasminbullock 
ParseGlob is equivalent to calling ParseFiles with the list of files matched by the pattern. 
So shouldn't we be doing `template.ParseGlob("*.html")`?

**Comment by benkasminbullock** [comment]
Sorry, I didn't notice that I had changed that line, it is as you say. I didn't mean to edit the code at all.

**Comment by thewhitetulip** [comment]
Can you change that line? Because I don't have write access on that branch, so we can merge it with the master branch

**Comment by benkasminbullock** [comment]
Is that OK now?

**Pull request merged**